README.md

```diff
@@ -1,24 +1,93 @@
 # reflect-node
 
-This is a client for the [Reflect API](https://reflect.io/docs/).
+This is a client for the [Reflect API](https://reflect.io/docs/reference/rest-api).
+
+This library includes support for [generating authentication tokens](#Generating-an-Authentication-Token)
+as well as [using the Reflect REST API](#API-methods).
 
 ## Installation
 
-This package is indexed in npm.
+Install reflect-node through npm:
 
 ```bash
 $ npm install reflect-node
 ```
 
+## API methods
+
+This functionality is currently under development. You can expect to see
+expanded support for our various endpoints down the road.
+
+### Getting started
+
+After [Installation](#Installation), require `reflect-node` from your
+application:
+
+```javascript
+const Reflect = require('reflect-node');
+```
+
+For the next step you'll need a Reflect API token. You can find yours by logging
+into https://app.reflect.io then navigating to the Tokens section of the Account
+page.
+
+```javascript
+const client = new Reflect.Client('my-api-token');
+```
+
+Once you've created a `Client` instance, you're ready to make authenticated
+requests to the Reflect API using `reflect-node`.
+
+The documentation below assumes you've completed these two steps, and have an
+instance of `Client` stored in a `client` variable.
+
+#### Resources
+
+`reflect-node` is broken up into various resources with methods for manipulating
+them. The various resources and their methods are documented below.
+
+##### Reporting
+
+Our reporting APIs are allow you to query the connections you've previously
+configured in Reflect.
+
+We currently only support the `report()` method. This method accepts four
+arguments:
+
+Name | Type | Description
+-----|------|--------------
+`connectionSlug` | string | The connection slug you wish to query
```

Comment:
This isn't the connection you want to query, but rather the data model (project) you want to query.